### PR TITLE
js-modern-syntax_destructuring-arrays done

### PR DESCRIPTION
**Pull request by PARAMET7**

### CODE REVIEW

File: 09-01-24/Destructuring-Arrays/js-modern-syntax_destructuring-arrays/index.js [hunk 1]
```diff
@@ -6,41 +6,59 @@ console.clear();
...
+export const [student1, [student2,student3], [student4,student5]] = nestedStudents;
+console.log(student1,student2,student3,student4,student5);
+
+
 // EXERCISE 6
 // Spread the values of the values array into the function call of the add function.
 // Keep in mind that you need to export the variable 'result' to make the test work.
 
-const values = [1, 6, 7, 9, 12, 5, 4];
+const values = [1, 6, 7, 9, 12, 5, 4,100];
```
Comment (jamoragar) [inline 09-01-24/Destructuring-Arrays/js-modern-syntax_destructuring-arrays/index.js:55]: Why did u add another number?
Btw, it is still working

**Review by jamoragar — APPROVED** [review]
Works fine to me!

File: 09-01-24/Destructuring-Arrays/js-modern-syntax_destructuring-arrays/index.js (hunk 1 above)
Comment (PARAMET7) [reply 09-01-24/Destructuring-Arrays/js-modern-syntax_destructuring-arrays/index.js:55]: just for checking,

Comment (PARAMET7) [reply 09-01-24/Destructuring-Arrays/js-modern-syntax_destructuring-arrays/index.js:55]: I know that they did ask for it, more for my understanding

**Pull request merged**